04_environment/index.tex

```diff
@@ -1 +1,400 @@
+\texttt{DisasterMitigate} & Mitigate the Disaster with the Amount of Resources in the Common Pool & - Effect of Disaster to each island\newline - Proportional effect of Disaster to each island (magnitude of disaster divided by the distance of the island from epicenter) & Resources in CP were exploited for the mitigation of the disaster and remaining effect has to be mitigated by each island \\ \hline
+\texttt{islandDeplete} & Deplete Islands after the disaster (with the amount of disaster that couldn't be mitigated by CP) & -Remaining effect for each Island\newline -List of Islands & The remaining proportional effect of the disaster to each island is causing the depletion of resources \\ \hline
+\end{tabular}
+\caption{\label{tab:table-name}Common Pool's main functions}
+\end{center}
+\end{table}
+
+The common pool has been computed as part of the infrastructure: with a threshold that is the amount of resources to hopefully help to completely mitigate the disaster. If the threshold is met, the damage from the disaster on the common pool and islands will be half the actual amount. This incentivises islands to contribute to the common pool and reflect the reality that, if there is enough preparation for the disaster, the damage will be much less compared to no preparation. Furthermore, it is worth mentioning that the threshold is a fixed value.\\
+
+Another important input to the implementation is the magnitude of the disaster’s damage to each island. As shown in Figure 1.9 , the disaster’s magnitude is 3.458. However, the disaster’s effect on individual islands do not add up to 3.458 because the storm’s epicenter is further away from the islands. This aspect creates a more realistic scenario to the game. From the islands’ individual damage, proportional effects are calculated to determine which island has been hit harder out of the 6 islands. The summation of these proportional effects is equal to 100\%, which is the total effective damage. \\
+
+If the common pool has enough resources to fully mitigate the disaster, the process is very easy. However, in the unfortunate events, the common pool resources will be used to mitigate each island's damage according to their proportional effect i.e., if an island is hit harder, it will receive more help from the common pool. This is also visualized in Figure 1.9, as island 4 withstands most damage from the storm, the common pool gave island 4 most help with 252.47 resources (“common pool mitigated” column). The updated damage value shows the leftover damage of the disaster that the common pool was not able to mitigate. These leftover damage will be used to further deplete the island's private pool of resources. If there are enough resources in the common pool totally mitigate the disaster then the update damage values will be 0.\\
```

Comment:
you cannot refer to the figures manually 

**check the guidelines !!!**

this is a rather serious mistake ...
@amertzani , @anhnguyen24

Reply:
agreed - this can cause really major problems and can be missed in the final report but is a big big deal - please reference figures correctly